backend/src/models/db/tables.js

```diff
@@ -125,9 +142,12 @@ function devInitDB(db) {
                 .forEach(item => { item.userID = userID })
 
             // insert course
-            return Promise.all(courseData.map(course =>
-                insertDB(db, 'course', { universityID, ...course })
-            ))
+            return Promise.all(
+                courseData.map(course => insertDB(db, 'course', { universityID, ...course }))
+                .concat(
```

Comment:
why `.concat`?

Reply:
Oh yeah actually Sorry. Maybe `...courseData.map(), ...subjectData.map()`

Reply:
yeah that's much better. donezo